Backend/Users/Service/Interfaces/IDoctorService.cs

```diff
@@ -10,5 +10,7 @@ public interface IDoctorService
         Doctor GetDoctorByRoomExaminationRoom(int roomId);
         Doctor UpdateDoctorDataBase(Doctor doctor);
         IEnumerable<Doctor> GetAll();
+        IEnumerable<Doctor> GetDoctorsBySpecialization(int specializationId);
+
```

Comment:
You can remove Specialization from GetDoctorBySpecialization, the parameter is self explanatory.

Reply:
GetDoctorsBy *** not GetDoctors